src/python/pants/reporting/streaming_workunit_handler.py

```diff
@@ -15,10 +15,17 @@
 class StreamingWorkunitContext:
     _scheduler: SchedulerSession
 
-    def digests_to_bytes(self, digests: Sequence[Digest]) -> Tuple[bytes]:
-        """Given a list of Digest objects, return a list of the bytes corresponding to each of those
-        Digests in sequence."""
-        return self._scheduler.digests_to_bytes(digests)
+    def single_file_digests_to_bytes(self, digests: Sequence[Digest]) -> Tuple[bytes]:
```

Comment:
Did you mean this?

```suggestion
    def single_file_digests_to_bytes(self, digests: Sequence[Digest]) -> Tuple[bytes, ...]:
```

Reply:
Bump. Why is this a `Tuple[bytes]`? Could we unpack it to be simply `bytes`? Will make for a cleaner API.